context/test_notebooks.py

```diff
@@ -22,17 +22,11 @@
 ]
 ignored_filenames = ["-csv", "benchmark", "target", "performance"]
 ignored_notebooks = [
-    'cusignal/api_guide/io_examples.ipynb', # 26gb data download
```

Comment:
We can remove this too. `cusignal` is no longer a part of RAPIDS:

* https://docs.rapids.ai/notices/rsn0032/
* https://github.com/rapidsai/cusignal (archived Septemebr 25, 2023)